### ISSUE TITLE
Fn to ignore Pad Analog Remap and allow passthrough

### ISSUE THREAD

**Issue by pgrison** [WootingKb/wootility-issues#272]
It would be nice to have an option for Analog Profiles below "Gamepad mapping override", to ignore the Analog mapping if Fn (or an another key) is pressed, and thus pass the underlying normal key.

Ie could be called "Keyboard key passthrough" or something like that.
(Obviously Fn-ed keys wouldn't be usable, or they would be become accessible through a Fn2 - but that would be a edge case)

Wootility v4.6.12
Wooting Two HE ARM
Firmware 2.7.3-beta.2

**Comment by simon-wh** [comment]
I'm pretty sure this behaviour could be replicated by just making your Fn layer the same as your main layer. Right now, when the Fn Layer is active, it shouldn't be inputting any gamepad mappings, as they are only on the main layer

**Comment by pgrison** [comment]
Yes but i have stuff on the Fn layer, so i must make a Fn layer specific for the analog profiles. (doable i think)
Could be easier to manage with a passthrough option though.
